controllers/exclude_processes.go

```diff
@@ -139,12 +155,42 @@ func (e excludeProcesses) reconcile(_ context.Context, r *FoundationDBClusterRec
 		}
 	}
 
+	var coordinatorExcluded bool
+	for _, excludeProcess := range fdbProcessesToExclude {
+		excludeString := excludeProcess.String()
+		_, excludedLocality := coordinatorsExclusionString[excludeString]
+		_, excludedAddress := coordinatorsAddress[excludeString]
```

Comment:
out of curiosity, is it possible to have one but not the other? e.g. could a coordinator ever be listed under status.ProcessGroup but not processGroup.Addresses?

Reply:
> e.g. could a coordinator ever be listed under status.ProcessGroup but not processGroup.Addresses?

The addresses are fetched from `status.ProcessGroup.Addresses`. We handle there the two different exclusions modes: Exclusion by IP (and port) or locality.